example/python/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ include(PythonDocstrings)
 set(WRAPPED_HEADERS ${CMAKE_SOURCE_DIR}/example.h)
 set(WRAPPING_SOURCES ${CMAKE_SOURCE_DIR}/python/example.cpp)
 
-docstrings(WRAPPING_SOURCES WRAPPED_HEADERS)
+docstrings(WRAPPING_SOURCES WRAPPED_HEADERS ${CMAKE_SOURCE_DIR})
```

Comment:
I don't know what this new parameter refers to, maybe consider giving it a name

Reply:
The parameter was already there, but the example was not updated when it was added. It has a name and it's documented in the macro declaration, so I'm not going to repeat that here.